resources/subscription-template-order-type.json

```diff
@@ -0,0 +1,68 @@
+{
+  "key": "subscription-template-order",
+  "name": {
+    "en": "Subscription template order"
+  },
+  "description": {
+    "en": "Subscription template order definition"
+  },
+  "resourceTypeIds": ["order"],
+  "fieldDefinitions": [
```

Comment:
Template order type needs to alway have also the fields of checkout-order since it needs to copy all the custom fields from the original order. This probably makes the type definition for the checkout-order obsolete. Or maybe even it makes the type definition for the template-order type obsolete since all the fields can has to be defined on the type used on checkout-order.